tests/e2e/README.md

```diff
@@ -111,6 +117,13 @@ tests require IBC logic.
 - `OSMOSIS_E2E_SKIP_CLEANUP` - when true, avoids cleaning up the e2e Docker
 containers.
 
+- `OSMOSIS_E2E_IS_FORK` - when true, utilizes fork logic over traditional upgrade logic
+
```

Comment:
```suggestion
```

Reply:
I think it was removed as per: https://github.com/osmosis-labs/osmosis/pull/1809/files#r905583441

Reply:
The link in my earlier comment was broken, updated now. Basically, the presence of `OSMOSIS_E2E_FORK_HEIGHT signifies whether this is a fork upgrade now